cmd/admin-bucket-handlers.go

```diff
@@ -169,6 +169,9 @@ func (a adminAPIHandlers) SetRemoteTargetHandler(w http.ResponseWriter, r *http.
 		writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErrWithErr(ErrAdminConfigBadJSON, err), r.URL)
 		return
 	}
+	if !target.Secure && port == "80" || target.Secure && port == "443" {
```

Comment:
This still enforces port to be specified for target.Endpoint @poornas - that is not necessary.

Reply:
actually `mc admin bucket remote add` fills in the port if unspecified - this is not required from command line

Reply:
So that is incorrect @poornas we should never manipulate them in this manner as it's not worth it.